Core/OfficeDevPnP.Core/Framework/Provisioning/ObjectHandlers/ObjectHandlerBase.cs

```diff
@@ -55,15 +55,26 @@ protected string Tokenize(string url, string webUrl)
                 {
                     return url.Substring(url.IndexOf("/_catalogs/masterpage", StringComparison.InvariantCultureIgnoreCase)).Replace("/_catalogs/masterpage", "{masterpagecatalog}");
                 }
-                Uri uri;
-                if(Uri.TryCreate(webUrl, UriKind.Absolute, out uri))
+                if (url.IndexOf(webUrl, StringComparison.InvariantCultureIgnoreCase) > -1)
```

Comment:
You're correct. I've changed it and committed a change where the logic is more simplified.